test/prepend.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+var throws = require('assert').throws;
+
+var errorEq = require('./utils').errorEq;
+var eq = require('./utils').eq;
+var S = require('..');
+
+
+describe('prepend', function() {
+
+  it('is a binary function', function() {
+    eq(typeof S.prepend, 'function');
+    eq(S.prepend.length, 2);
+  });
+
+  it('type checks its arguments', function() {
+    throws(function() { S.prepend('a', 'bc'); },
```

Comment:
Could we test the type checking of the final argument by adding a test that uses the placeholder as the first arg?